packages/flex-plugin-scripts/src/clients/services.ts

```diff
@@ -0,0 +1,41 @@
+import { AuthConfig } from './auth';
+import BaseClient from './baseClient';
+import { Service, ServiceResource } from './serverless-types';
+
+export default class ServiceClient extends BaseClient {
+  public static getBaseUrl = (): string => {
+    const realm = process.env.TWILIO_SERVERLESS_REALM;
+    if (!realm) {
+      return 'https://serverless.twilio.com/v1';
+    }
+
+    return `https://serverless.${realm.toLowerCase()}.twilio.com/v1`;
+  }
+
+  constructor(auth: AuthConfig) {
+    super(auth, ServiceClient.getBaseUrl());
+  }
+
+  /**
+   * Fetches the default {@ Service}
+   */
+  public getDefault = (): Promise<Service> => {
+    return this.list()
+      .then((resource) => resource.services.find((s) => s.unique_name === 'default'))
+      .then((service) => {
+        if (!service) {
+          throw new Error('No Runtime service named default was found.');
```

Comment:
can we be consistent on whether we capitalize "service" or not?

Reply:
Lowercase, I'll try to find all instances and lowercase them.

Reply:
👍